src/meta/src/hummock/compaction/base_level_compaction_picker.rs

```diff
@@ -132,21 +124,15 @@ impl CompactionPicker for LevelCompactionPicker {
             if write_amplification < all_level_amplification {
                 return None;
             }
+            // reverse because the ix of low sub-level is smaller.
+            input_levels.reverse();
```

Comment:
A thought, do we need to add some functions to manage the logic corresponding to sub_level. Like the `reverse`, `sub_levels overlap type` and so on, the logic of other Levels is still a bit inconsistent

Reply:
Seems difficult. Maybe you can give another PR to fix it.